Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling.yml

```diff
@@ -628,14 +716,18 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      id: aef83a49-1bea-422e-8403-cadf27802149
+      id: 415a1b04-7375-4189-82dd-48f9b54d4ff9
       iscommand: false
       name: Final Closure
       type: title
       version: -1
       description: ''
-    taskid: aef83a49-1bea-422e-8403-cadf27802149
-    timertriggers: []
+    taskid: 415a1b04-7375-4189-82dd-48f9b54d4ff9
+    timertriggers:
+    - fieldname: containmentsla
+      action: stop
+    - fieldname: remediationsla
+      action: stop
```

Comment:
Should the containment and remediation stop at the same time?

Reply:
It was the best option without adding an additional label/header section there. Did you see the image?

Reply:
As we discussed, we are keeping these stop timers, moving the remediation timer to be on top of the containment start timer.